### ISSUE TITLE
Handle ABBYY failures due to bad images

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/common-accessioning#1213]

**Comment by dnoneill** [comment]
I believe this is already in place with the logic @thatbudakguy added. We should probably test to make sure this is true but I don't think there is any implementation here.

**Comment by thatbudakguy** [comment]
Yeah, as long as ABBYY sets the `IsFailed` attribute in the result XML, we should catch it as a failure and mark the `ocr-create` step as failed. We should also send the error messages coming from ABBYY to be viewable in Argo.

Agree it would be good to confirm this works as expected before closing this ticket.

**Comment by jmartin-sul** [comment]
@andrewjbtw accessioned some things in the failed queue, and will add links to them, for use as fixture/test objects.

**Comment by andrewjbtw** [comment]
This search in Argo-QA will bring up 3 sets of images that were recently in the production ABBYY Exceptions folder: https://argo-qa.stanford.edu/catalog?f%5Btag_ssim%5D%5B%5D=ABBYY+%3A+exception+example
